webhooks-extension/pkg/endpoints/shared-test-funcs.go

```diff
@@ -35,74 +35,59 @@ func dummyK8sClientset() *fakek8sclientset.Clientset {
 		// get the runtime.Object
 		obj := action.(k8stesting.CreateAction).GetObject()
 		// convert to unstructured map[string]interface{}
 		unstructuredObj, err := runtime.DefaultUnstructuredConverter.ToUnstructured(obj)
 		if err != nil {
-			newErr := fmt.Errorf("ERROR : Problem with githubsource creation in shared-test-funcs.go")
+			newErr := fmt.Errorf("ERROR : Problem with taskrun creation in shared-test-funcs.go")
 			return false, nil, newErr
 		}
-		// convert to GitHubSource
-		var result githubsource.GitHubSource
+		// convert to TaskRun
+		var result pipelinesv1alpha1.TaskRun
 		mapstructure.Decode(unstructuredObj, &result)
 		// store to end of map
-		index := len(sources) + 1
+		index := len(taskruns) + 1
 		result.Name = fmt.Sprintf("tekton-%d", index)
-		sources[result.Name] = result
+		// HARD CODING TO SUCCESSFUL TASKRUN
+		condition := &apis.Condition{
+			Type:   apis.ConditionSucceeded,
+			Status: "True",
+		}
+		result.Status.SetCondition(condition)
+		taskruns[result.Name] = result
 
 		//In case debug needed, you might want to uncomment this
-		//fmt.Printf("Sources: %+v", sources)
-		return true, &result, nil
-	})
-
-	// Need to intercept calls to delete githubsource, so that we can return the githubsource from the map we created
-	// in the above code.
-	resultClient.Fake.PrependReactor("delete", "githubsources", func(action k8stesting.Action) (handled bool, ret runtime.Object, err error) {
-		srcName := action.(k8stesting.DeleteAction).GetName()
-		delete(sources, srcName)
-		result := sources[srcName]
-
+		//fmt.Printf("taskruns: %+v", taskruns)
 		return true, &result, nil
 	})
 
 	// Need to intercept calls to get githubsource, so that we can return the githubsource from the map we created
 	// in the above code.
-	resultClient.Fake.PrependReactor("get", "githubsources", func(action k8stesting.Action) (handled bool, ret runtime.Object, err error) {
+	resultClient.Fake.PrependReactor("get", "taskruns", func(action k8stesting.Action) (handled bool, ret runtime.Object, err error) {
 		srcName := action.(k8stesting.GetAction).GetName()
-		if ghs, found := sources[srcName]; found {
-			return true, &ghs, nil
+		if tr, found := taskruns[srcName]; found {
+			return true, &tr, nil
 		}
-
-		return true, &githubsource.GitHubSource{}, errors.New("githubsource not found")
-	})
-
-	// Need to intercept calls to list githubsource, so that we can list the githubsources from the map we created
-	// in the above code.
-	resultClient.Fake.PrependReactor("list", "githubsources", func(action k8stesting.Action) (handled bool, ret runtime.Object, err error) {
-		sourceArray := []githubsource.GitHubSource{}
-		for _, source := range sources {
-			sourceArray = append(sourceArray, source)
-		}
-		list := githubsource.GitHubSourceList{Items: sourceArray}
-		return true, &list, nil
+		fmt.Println("-------------------HI DUANE")
```

Comment:
Hello

Reply:
Sorry - was clearly talking to Duane .... but Hi Adam